arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTraceViewController.swift

```diff
@@ -0,0 +1,397 @@
+            switch Switches.allCases[indexPath.row] {
+            case .barriers:
+                cell.accessoryView = barriersSwitch
+                cell.textLabel?.text = Switches.barriers.label
+            case .containers:
+                cell.accessoryView = containersSwitch
+                cell.textLabel?.text = Switches.containers.label
+            }
+            return cell
+        case .conditions:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "ConditionCell", for: indexPath)
+            cell.textLabel?.text = expressionToString(expression: traceConditionalExpressions[indexPath.row])
+            return cell
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        if indexPath.section == Section.conditions.rawValue && indexPath.row != 0 {
+            // Enable editing for the rest of conditions section.
+            return true
+        } else {
+            // Disable editing for the first row of conditions section and other rows.
+            return false
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if indexPath.section == Section.conditions.rawValue && editingStyle == .delete {
+            traceConditionalExpressions.remove(at: indexPath.row)
+            tableView.deleteRows(at: [indexPath], with: .fade)
+            // Reload footer label.
+            chainedConditionsLabel.text = expressionString
+        }
+    }
+}
+
+private extension AGSUtilityCategoryComparisonOperator {
+    /// An extension of `AGSUtilityCategoryComparisonOperator` that returns a human readable description.
+    /// - Note: You may also create a `AGSUtilityCategoryComparison` with
+    ///         `AGSUtilityNetworkDefinition.categories` and `AGSUtilityCategoryComparisonOperator`.
+    var title: String {
+        switch self {
+        case .exists: return "Exists"
+        case .doesNotExist: return "DoesNotExist"
+        @unknown default: return "Unknown"
+        }
+    }
+}
+
+private extension AGSUtilityAttributeComparisonOperator {
+    /// An extension of `AGSUtilityAttributeComparisonOperator` that returns a human readable description.
+    var title: String {
+        switch self {
+        case .equal: return "Equal"
+        case .notEqual: return "NotEqual"
+        case .greaterThan: return "GreaterThan"
+        case .greaterThanEqual: return "GreaterThanEqual"
+        case .lessThan: return "LessThan"
+        case .lessThanEqual: return "LessThanEqual"
+        case .includesTheValues: return "IncludesTheValues"
+        case .doesNotIncludeTheValues: return "DoesNotIncludeTheValues"
+        case .includesAny: return "IncludesAny"
+        case .doesNotIncludeAny: return "DoesNotIncludeAny"
+        @unknown default: return "Unknown"
+        }
+    }
+}
```

Comment:
I think it is fine to keep the title string like this, as these strings serve as the comparison operators in the trace condition expressions. They are not intended to read naturally.